iocage_lib/ioc_exceptions.py

```diff
@@ -64,3 +65,25 @@ class JailMissingConfiguration(JailMisconfigured):
 
 class ValidationFailed(ExceptionWithMsg):
     pass
+
+
+@contextmanager
+def ignore_exceptions(
+        *exceptions, action=None, action_args=None,
```

Comment:
action is a confusing name IMHO. The only use cases you have so far are static return values, and thats what this really is, a `return_value`

If a callable is a edge cage perhaps it makes sense to only use the callable and no args and kwargs. These could be passed using functools.partial if needed.